core/src/main/scala/org/apache/spark/internal/plugin/PluginContainer.scala

```diff
@@ -133,17 +140,27 @@ object PluginContainer {
 
   val EXTRA_CONF_PREFIX = "spark.plugins.internal.conf."
 
-  def apply(sc: SparkContext): Option[PluginContainer] = PluginContainer(Left(sc))
+  def apply(
+      sc: SparkContext,
+      resources: java.util.Map[String, ResourceInformation]
+  ): Option[PluginContainer] = PluginContainer(Left(sc), resources)
```

Comment:
indentation?

Reply:
hmm, indentation is correct by IntelliJ scalastyle, I've seen it both ways, I'll change it though as I think this is less used.